app/reducers/MomentReducer.js

```diff
@@ -0,0 +1,19 @@
+import { MOMENT_ACTIONS } from '../constants';
+import SessionUtils from '../utils/SessionStorage';
+import update from 'react-addons-update';
+
+class MomentReducer {
```

Comment:
Thanks Sir. One question though, Is there a performance implication(+ve) with this approach? 
I'm guessing that having to import only specific methods from a file reduces the memory footprint and hence speed. Is that correct?